services/distribution/src/main/java/app/coronawarn/server/services/distribution/config/DistributionServiceConfig.java

```diff
@@ -1603,6 +1603,8 @@ public void setRequireEvaluationTypeHardwareBacked(Boolean requireEvaluationType
 
       private DgcTestCertificateParameters dgcTestCertificateParameters;
 
+      private Integer expirationThresholdInDays;
```

Comment:
@mlenkeit - should we define some `@Min(0)` or `@Max(100)` values?

Reply:
@hilmarf  Good idea!

Reply:
which values make most sense?